docs/ReleaseNotes.html

```diff
@@ -82,6 +82,26 @@ <h1 align="center">Password Safe 3.39.2 Release Notes ???? 2016</h1>
 <td>Empty groups are now added when merging one PasswordSafe database into another.</td>
 </tr>
 <tr>
+<td align="center" width="100">[]</td>
+<td>List view column widths are no longer reset when dragging columns to different
+positions or columns are added or removed.</td>
+</tr>
+<tr>
+<td align="center" width="100">[]</td>
+<td>The user specified Notes font is now also applied to the Notes column in the
+List view.</td>
+</tr>
+<tr>
+<td align="center" width="100">[]</td>
+<td>Entry Protected sttaus and AutoType string is now correclty displayed in the

```

Comment:
Should be
"Entry Protected state and AutoType string are now correctly displayed in the"